### DISCUSSION TITLE
Duplicate grammarly casks

### DISCUSSION THREAD

**Discussion by arozbiz** [Homebrew/discussions#5111] [Casks]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.2.7
ORIGIN: https://github.com/Homebrew/brew
HEAD: 6170b6297c61ef0b68814c708eeff2199d8b0d11
Last commit: 3 days ago
Core tap JSON: 05 Feb 21:28 UTC
Core cask tap JSON: 05 Feb 21:28 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 3.1.4 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.1.4/bin/ruby
CPU: octa-core 64-bit arm_blizzard_avalanche
Clang: 15.0.0 build 1500
Git: 2.39.3 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 8.4.0 => /usr/bin/curl
macOS: 14.3-arm64
CLT: 15.1.0.0.1.1700200546
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

The [grammarly](https://github.com/Homebrew/homebrew-cask/blob/4cbfdc267867b0e4e00033f3573af076cc68ca9c/Casks/g/grammarly.rb) and [grammarly-desktop](https://github.com/Homebrew/homebrew-cask/blob/4cbfdc267867b0e4e00033f3573af076cc68ca9c/Casks/g/grammarly-desktop.rb) casks seem like duplicates, with the grammarly-desktop cask being the most recently updated one.

### Relevant casks

grammarly, grammarly-desktop

**Comment by SMillerDev** [comment]
They download different files, so it doesn't seem that way to me.

**Comment by arozbiz** [comment]
But they're installing the same app "Grammarly Desktop"---shouldn't there only be one cask for doing this? To be clear, I'm new to Homebrew, so apologies if I'm missing something.

**Reply by SMillerDev** [reply]
One claims to be installing "Grammarly Editor" and the other "Grammarly Desktop". What makes you say they both install the same thing?
